src/main/java/org/opensrp/connector/openmrs/schedule/OpenmrsSyncerListener.java

```diff
@@ -48,16 +43,11 @@
 	
 	private final ClientService clientService;
 	
-	@Autowired
-	private Dhis2TrackCaptureConnector dhis2TrackCaptureConnector;
-	
 	@Autowired
 	public OpenmrsSyncerListener(ScheduleService opensrpScheduleService, ActionService actionService, ConfigService config,
 	    ErrorTraceService errorTraceService, PatientService patientService, EncounterService encounterService,
 	    ClientService clientService, EventService eventService) {
 		
-		this.opensrpScheduleService = opensrpScheduleService;
-		this.actionService = actionService;
```

Comment:
Why did you remove this and still left `opensrpScheduleService` & `actionService` as method params?

Reply:
These are **private** fields which are unused anywhere in this class hence they are not required